src/fragment-generation-utils.js

```diff
@@ -1257,7 +1257,7 @@ const makeWalkerForNode = (node, endNode) => {
 
   const walker = document.createTreeWalker(
       blockAncestor, NodeFilter.SHOW_ELEMENT | NodeFilter.SHOW_TEXT, (node) => {
-        return fragments.internal.filterFunction(node);
+        return fragments.internal.visibleNodesInRangeFilterFunction(node);
```

Comment:
`filterFunction` was a pretty bad name to begin with (it sounds like it should be an object). While you're at it, why not change this to be more "verb-like"? Something like `isNodeVisibleInRange`

Reply:
My only concern with `isNodeVisibleInRange` is that it sounds like it returns Boolean instead of a NodeFilter constant. Maybe `acceptNodeIfVisibleInRange`? I don't love it but that's pretty much what the function does.

Reply:
Ah, good catch! Your suggestion is fine.

Reply:
Thanks! I'll rename and update the PR.